pom.xml

```diff
@@ -400,6 +400,41 @@
                 </configuration>
             </plugin>
         </plugins>
+        <pluginManagement>
```

Comment:
It this change necessary?
What does it do?
Currently, we do not want any eclipse specific settings in our pom?

Reply:
Yes, you are right... I have committed them by mistake. Will remove it